src/ribasim_nl/ribasim_nl/reset_index.py

```diff
@@ -0,0 +1,50 @@
+# %%
+import pandas as pd
+from ribasim import Model
+
+from ribasim_nl.model import TABLES, get_table
+
+
+def reset_index(model: Model, node_start=1):
```

Comment:
I hope this won't be needed after we have https://github.com/Deltares/Ribasim/pull/1110

Reply:
Maybe, but let's see. A bit worried about `users won't have to think about node-ids at all`. I think advanced users, e.g. having to merge multiple (different style) models will ultimately have to work around ribasim-Python to get things fixed within deadline :-).